CMakeLists.txt

```diff
@@ -23,7 +25,7 @@ if(MSVC)
 	endif()
 elseif(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
 	# Update if necessary
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -Wno-reorder -pedantic ")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -Wno-reorder -pedantic ")
```

Comment:
replaces a tab with spaces.
1. the highlight on the whitespace is a giveaway
2. when marking the whitespace under the highlight on the left you can only mark the whole block (tab), on the right you can mark single spaces